activestorage/test/test_helper.rb

```diff
@@ -0,0 +1,54 @@
+require File.expand_path("../../test/dummy/config/environment.rb", __FILE__)
+
+require "bundler/setup"
+require "active_support"
+require "active_support/test_case"
+require "active_support/testing/autorun"
+require "byebug"
+
+require "active_job"
+ActiveJob::Base.queue_adapter = :test
+ActiveJob::Base.logger = nil
+
+require "active_storage"
+
+require "yaml"
+SERVICE_CONFIGURATIONS = begin
+  YAML.load(ERB.new(Pathname.new(File.expand_path("../service/configurations.yml", __FILE__)).read).result).deep_symbolize_keys
```

Comment:
If you have a "service/configurations.yml" file, but every single line is commented out, then an error occurs when running tests:

```
git:active-storage-import~/code/rails/activestorage$ rake
~/code/rails/activestorage/test/test_helper.rb:17:in `<top (required)>': undefined method `deep_symbolize_keys' for false:FalseClass (NoMethodError)
	from ~/code/rails/activestorage/test/controllers/direct_uploads_controller_test.rb:1:in `require'
```

The reason is that `YAML.load(..an empty file content..)` simply returns `false`, and not `{}`.

If you think this should be fixed, you can merge this:
https://github.com/rails/rails/compare/active-storage-import...claudiob:only-disk?expand=1

I haven't explicitly opened a PR because it's a commit against a branch different than `master`, so I wasn't sure if that was the right approach.